src/web/server/render.ts

```diff
@@ -67,8 +67,10 @@ export const render = (
 
 		res.status(200).send(resp);
 	} catch (e) {
-		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-		res.status(500).send(`<pre>${e.stack}</pre>`);
+		const error = e as Error;
+		res.status(500).send(
+			`<pre>${error.stack || `Unknown error: ${error.message}`}</pre>`,
```

Comment:
I'm not sure we should be prefixing potentially reader facing error messages in the code